framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -579,7 +579,7 @@ def set_integrator_params(self, **data):
         rendering_rules = SimulatorFragmentRenderingRules(None, None, SimulatorWizzardURLs.SET_INTEGRATOR_PARAMS_URL,
                                                           is_simulator_copy, is_simulator_load,
                                                           self.last_loaded_form_url, cherrypy.request.method,
-                                                          is_noise_fragment=True)
+                                                          is_noise_fragment=False)
```

Comment:
Indeed, this is a flag I have missed when refactoring the code. Thanks for catching it!